### PR TITLE
implement OAM layer services

### PR DESCRIPTION
**Pull request by at00825957**
implement OAM layer services

### CODE REVIEW

File: server/service/individualServices/ElasticsearchPreparation.js [hunk 1]
```diff
@@ -1,48 +1,76 @@
 const { elasticsearchService, getIndexAliasAsync, operationalStateEnum } = require('onf-core-model-ap/applicationPattern/services/ElasticsearchService');
+const logicalTerminationPoint = require('onf-core-model-ap/applicationPattern/onfModel/models/LogicalTerminationPoint');
```
Comment (PrathibaJee) [inline server/service/individualServices/ElasticsearchPreparation.js:2]: I can see that this file is copied from ApplicationLayerTopology. Kindly please revert this file to original state.

**Review by PrathibaJee — CHANGES_REQUESTED** [review]
Please incorporate the review comments. Thanks.

File: server/service/StringProfileService.js [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
-      resolve(examples[Object.keys(examples)[0]]);
-    } else {
-      resolve();
+exports.getStringProfileEnumeration = function(url) {
+  return new Promise(async function (resolve, reject) {
+    try {
+      var value = await fileOperation.readFromDatabaseAsync(url);
+      var response = {};
+      response['application/json'] = {
+        "http-client-interface-1-0:release-number": value
```
[the remark this answers is not in the document]
Comment (at00825957) [reply server/service/StringProfileService.js]: Resolved

File: server/service/StringProfileService.js [hunk 3]
```diff
@@ -28,16 +34,21 @@ exports.getStringProfileEnumeration = function(uuid) {
...
-      resolve(examples[Object.keys(examples)[0]]);
-    } else {
-      resolve();
+exports.getStringProfilePattern = function(url) {
+  return new Promise(async function (resolve, reject) {
+    try {
+      var value = await fileOperation.readFromDatabaseAsync(url);
+      var response = {};
+      response['application/json'] = {
+        "string-profile-1-0:pattern": value
```
[the remark this answers is not in the document]
Comment (at00825957) [reply server/service/StringProfileService.js]: Resolved

File: server/service/individualServices/ElasticsearchPreparation.js (hunk 1 above)
Comment (at00825957) [reply server/service/individualServices/ElasticsearchPreparation.js]: Resolved

**Review by PrathibaJee — APPROVED** [review]
Thanks for incorporating the review comments.

**Pull request merged**